src/framework/components/screen/component.js

```diff
@@ -19,28 +66,42 @@ pc.extend(pc, function () {
     var _transform = new pc.Mat4();
 
     pc.extend(ScreenComponent.prototype, {
-        // used for debug rendering
-        update: function (dt) {
-            // var p = this.entity.getPosition();
-            // var s = this.entity.getLocalScale();
-            // var r = this.entity.right.clone().scale(this._resolution.x * s.x/2);
-            // var u = this.entity.up.clone().scale(this._resolution.y * s.y/2);
-
-            // var corners = [
-            //     p.clone().sub(r).sub(u),
-            //     p.clone().sub(r).add(u),
-            //     p.clone().add(r).add(u),
-            //     p.clone().add(r).sub(u)
-            // ];
-
-            // var points = [
-            //     corners[0], corners[1],
-            //     corners[1], corners[2],
-            //     corners[2], corners[3],
-            //     corners[3], corners[0]
-            // ];
-
-            // this.system.app.renderLines(points, new pc.Color(1,1,1));
+
+        // Draws a debug box transforming local-spaced corners using current transformation matrix.
+        // Helps to understand where the bounds of the screen really are.
+        _drawDebugBox: function (dt) {
```

Comment:
Debugging code, is something that should not end up in Production code. There is #define we use to specify if it should be used with debug compiled engine. Consider adding this flag, so this debugging method does not end up being in release engine.